tools/bioformats2raw/bf2raw.xml

```diff
@@ -0,0 +1,127 @@
+<tool id="bf2raw" name="bioformats2raw" version="0.0.1">
+    <description>A wrapper for the bioformats2raw package that converts various image file formats to the cloud-optimised OME-Zarr format following the OME-NGFF specification</description>
```

Comment:
According to https://github.com/elixir-europe/biohackathon-projects-2023/blob/main/16/conventions.md, maybe:
```suggestion
    <description>with Bioformats</description>
```
or something similar?

Please also add the annotations.

Reply:
This is done now.